core-test/src/test/scala/org/bitcoins/core/wallet/builder/BitcoinTxBuilderTest.scala

```diff
@@ -387,6 +387,36 @@ class BitcoinTxBuilderTest extends BitcoinSAsyncTest {
     }
   }
 
+  it must "succeed to sign a cltv spk that uses a second-based locktime" in {
```

Comment:
should we add test cases for block height as well? Do you think negative test cases are necessary to?